@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -874,6 +869,24 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         const d = cameraToOrigin.length();
         let farPlane: number;
         if (camera instanceof THREE.PerspectiveCamera) {
+            const halfVerticalFovAngle = THREE.MathUtils.degToRad(camera.fov / 2);
+
+            // Ratio of the height of the camera compared to the distance between the center of the
+            // screen and the middle of the top of the screen (here set to be value 1). We just need
+            // this to be a ratio because we are interested in just computing the direction of the
+            // upper right corner and not any specific length.
+            const height = 1 / Math.tan(halfVerticalFovAngle);
```

Comment:
It took me a while to understand what you meant by height here. Since you are making a vector in camera space, height sounds related to camera y axis, not z axis like here.
I'd call it 'distance' or 'z'. The height here would be 1.

Reply:
Good point, I will rename.